locale/en/blog/wg/diag-wg-update-2017-02.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Diag WG Update - Many new tools, phasing out some old ones
+author: Josh Gavant (@joshgav)
+date: 2017-02-28T08:00:00.000Z
+category: wg
+status: publish
+slug: diag-wg-update-2017-02
+layout: blog-post.hbs
+---
+
+In surveys and discussions with Node.js users we consistently hear of your need
+for better tools and APIs for debugging and understanding your modules and apps.
+In fact, in last year's [Node.js Foundation survey][], the biggest ask from
+developers and technical leads was for better tools.
+
+[Node.js Foundation survey]: https://nodejs.org/static/documents/2016-survey-report.pdf
+
+Based on that feedback and thanks to contributions and collaboration from across
+our community, over the past year several experimental diagnostic APIs and tools
+have landed in Node itself and the Node.js Foundation, including:
+
+* [async_hooks][] for sharing context across async boundaries
+* [Inspector][] and [node-inspect][] for stepping and profiling 
+* [Trace Controller][] for gathering and streaming traces from V8 and Node
+* [node-report][] and [llnode][] for post-mortem analysis
+
+[async_hooks]: https://github.com/nodejs/node/pull/8531
+[Inspector]: https://github.com/nodejs/node/issues?utf8=%E2%9C%93&q=label%3Ainspector%20
+[node-inspect]: https://github.com/nodejs/node-inspect
+[Trace Controller]: https://github.com/nodejs/node/pull/9304
+[node-report]: https://github.com/nodejs/node-report
+[llnode]: https://github.com/nodejs/llnode
+
+Over the coming months we hope to graduate these from experimental status with
```

Comment:
"experimental" might be confused with the ["Stability Index - Experimental"](https://nodejs.org/dist/latest-v7.x/docs/api/documentation.html#documentation_stability_index). Some of these tools like `async_hooks` will likely graduate to "Stability Index - Experimental" from a pre-experimental state.